crates/sui-core/src/authority.rs

```diff
@@ -3488,7 +3488,10 @@ impl AuthorityState {
             )
             .await
             .tap_ok(|_| {
-                debug!("commit_certificate finished");
+                debug!(
+                    effects_digest = ?effects.digest(),
+                    "commit_certificate finished"
```

Comment:
i added the effects to this log while debugging and it seems useful for quickly checking for forks in logs